packages/web/e2e/README.md

```diff
@@ -0,0 +1,61 @@
+# @osmosis-labs/web/e2e
+
+This package contains the Playwright E2E tests and a Keplr wallet
+
+## Environment Variables
+
+By default, configuration is pointing to the [Stage](https://stage.osmosis.zone) environment. Tests will automatically install a wallet. All you need to add is a privat keys for wallets that are being used:
+
+- Frontend tests are using `osmo1ka7q9tykdundaanr07taz3zpt5k72c0ut5r4xa` wallet
+- Monitoring tests are using `osmo1dkmsds5j6q9l9lv4dkhas68767tlqfx8ls5j0c` and `osmo1fapvfx64af2eperkggnwd6zmpzdvvnq4xjc2dv` wallets
+- example command `export PRIVATE_KEY=0x....` for the frontend tests.
+
+### Run E2E Tests
+
+To install Playwright, please execute `npx playwright install --with-deps chromium` from the /web folder.
+
+To run a Select pair tests, please execute `npx playwright test -g "Test Select Swap Pair feature"` from the /web folder.
+To run a Swap E2E tests, please execute `npx playwright test -g "Test Swap feature"` from the /web folder.
+To run a Monitoring E2E tests, please execute `npx playwright test monitoring --timeout 180000` from the /web folder.
+
+Tests can be executed locally in a browser by changing `headless: true` to `headless: false`.
+
+In CI secrets frontend test privet key is referenced as:
```

Comment:
typo: `privet`

Reply:
fixed, thx. privet is actually a hi in Russian :)